mypy/stubgen.py

```diff
@@ -1385,6 +1481,9 @@ def parse_options(args: List[str]) -> Options:
     parser.add_argument('--include-private', action='store_true',
                         help="generate stubs for objects and members considered private "
                              "(single leading underscore and no trailing underscores)")
+    parser.add_argument('--export-less', action='store_true',
+                        help=("don't implicitly export all names imported from other modules "
+                              "in the same package"))
```

Comment:
Maybe update the docs with the new flag? (In a separate PR if you prefer.)

Reply:
A good idea. I'll create a separate PR.